### PR TITLE
Bug 1193498 - [Music] Opening a downloaded song doesn't work

### PR DESCRIPTION
**Pull request by jimporter**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4fc19cc32628fad6a116bdefb80d026a7187b975

File: apps/music/js/open.js [hunk 1]
```diff
@@ -37,14 +37,18 @@ function handleOpenActivity(request) {
     PlayerView.init(TYPE_SINGLE);
     PlayerView.stop();
 
-    AudioMetadata.parse(blob).then((metadata) => {
+    PlaybackQueue.loadSettings().then(() => {
+      return AudioMetadata.parse(blob);
+    }).then((metadata) => {
+      console.warn(blob.name);
```
Comment (justindarc) [inline apps/music/js/open.js]: Was this accidentally left in?

Comment (jimporter) [inline apps/music/js/open.js]: Yep! I'll remove that before landing.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=5f5996f4e97680605b62aa1287c4f70469992cd4

**Pull request merged**